action-scripts/brownie/scripts/report_gauges.py

```diff
@@ -188,13 +200,20 @@ def _parse_removed_transaction(transaction: dict, **kwargs) -> Optional[dict]:
     pool_name, pool_symbol, pool_id, pool_address, a_factor, fee, style = _extract_pool(
         chain, gauge, gauge_selectors
     )
+    addr =AddrBook("mainnet")
+    to = transaction['to']
+    to_name = addr.reversebook.get(to, "!!NOT-FOUND")
+    if to_name == "20221124-authorizer-adaptor-entrypoint/AuthorizerAdaptorEntrypoint":
+        to_string = "AAEntrypoint"
+    elif isinstance(to_name, str):
+        to_string = f"!!f{to_name}??"
```

Comment:
Siggestion to ,ake it `NOT_FOUND` or something

Reply:
same